fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,230 @@
+        for k, v in data.items():
+            if type(v) in [list]:
+                data[k] = str(v)
+        return data
+    if isinstance(input_dict["data"], list):
+        for element in input_dict["data"]:
+            element["athlete_id"] = input_dict["athlete_id"]
+            element["activity_id"] = input_dict["activity_id"]
+            for k, v in element.items():
+                if type(v) in [list]:
+                    element[k] = str(v)
+        return input_dict["data"]
```

Ruff (0.8.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)